Documentation/Reference/GuidesXml.rst

```diff
@@ -9,19 +9,33 @@ Configuration of the rendering - guides.xml
 This XML file contains meta information and configuration used during rendering
 of a manual.
 
-..  rubric:: Example:
-
 ..  literalinclude:: _guides-simple.xml
     :caption: Documentation/guides.xml
 
+..  contents:: Table of Contents:
+    :local:
+
+..  _guides-xml-init:
+
+Init new guides.xml
```

Comment:
I would reword this, what is guides.xml? 

`initialize new documentation project`

Reply:
We are on the page, which describes what the guides.xml does here. Therefore I think in this context it is ok